runatlantis.io/docs/server-configuration.md

```diff
@@ -802,6 +802,14 @@ This is useful when you have many projects and want to keep the pull request cle
   * Allowlist all repositories
     * `--repo-allowlist='*'`
 
+### `--set-atlantis-apply-check-successful-if-no-changes`
+  ```bash
+  atlantis server --set-atlantis-apply-check-successful-if-no-changes
+  # or
+  ATLANTIS_SET_ATLANTIS_APPLY_CHECK_SUCCESSFUL_IF_NO_CHANGES=true
+  ```
+  `--set-atlantis-apply-check-successful-if-no-changes` will set the `atlantis/apply` status check to "passing" on a VCS pull request if the `atlantis plan` command results in "No Changes". Defaults to `false`.
```

Comment:
Please describe the use-case for this command so devs know when to enable this flag